cfpb/consumerfinance.gov
# Move Ask-cfpb and job listing assets to apps

Ask has some JS/CSS that is spread out across our global styles, but really are Ask-specific. This moves those assets to a new ask-cfpb app.

This PR also moves the job listing assets to a `careers` app, excluding the one on-demand CSS file.

## Changes

- Move Ask-cfpb assets to an app.
- Move careers job listing assets to an app.


## How to test this PR

Ask landing - search should show autocomplete
http://localhost:8000/ask-cfpb/

Ask landing Spanish - search show autocomplete
http://localhost:8000/es/obtener-respuestas/

Ask search - search should not show autocomplete
http://localhost:8000/ask-cfpb/search

Ask answers - search does not have an autocomplete
https://www.consumerfinance.gov/consumer-tools/auto-loans/answers/

Summaries:

Ask detail - resize to mobile to see summary component
http://localhost:8000/ask-cfpb/how-can-i-tell-if-a-friend-neighbor-or-family-member-is-a-victim-of-financial-exploitation-en-1933/

Job listing - click on a current job posting and resize to mobile to see summary component
http://localhost:8000/about-us/careers/current-openings/ 

Prepaid agreements - minimal summary should be present
http://localhost:8000/data-research/prepaid-accounts/search-agreements/detail/48439/

## Notes and todos

- In Ask, the summary.js is only used on Ask details view, but for simplicity the JS is delivered to all ask pages.
